api/src/main/java/com/redhat/insights/http/BackoffWrapper.java

```diff
@@ -53,6 +59,13 @@ public int run() {
         action.run();
         return count;
       } catch (Throwable err) {
+        if (err instanceof FatalBackoffException) {
+          InsightsException fatalErr =
+              new InsightsException(
+                  ERROR_CLIENT_BACKOFF_RETRIES_FAILED, "Exponential backoff fatal error");
+          fatalErr.addSuppressed(err);
```

Comment:
Why is this addSuppressed() rather than a regular "caused by"?

Reply:
Sorry this code was _NOT_ supposed to be there!